docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/README.md

```diff
@@ -0,0 +1,140 @@
+# {{ cookiecutter.app_verbose_name }}
+
+## Cookiecutter Notes
+
+### Preparing for Docker
+
+Before building the Docker container, initialize the database:
+
+```bash
+docker-compose run --rm app python manage.py makemigrations
```

Comment:
I think that's because .gitignore (reasonably) excludes .env files from the repo. I'll go back to including instructions to copy over an example .env file.